tests_integration/test_package.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from pathlib import Path
+
+from unittest.mock import MagicMock, patch
+from tests_integration.snowflake_connector import snowflake_session
+from tests_integration.test_utils import row_from_cursor
+
+import pytest
+
+
+class TestPackage:
+    DATABASE_NAME = "JSIKORSKI_DB"
+    SCHEMA_NAME = "TEST_SCHEMA"
+    STAGE_NAME = "JSTS_2"  # TODO change this to generated one in the future
```

Comment:
Stage name can be unique in database scope, don't have to be generated

Reply:
Yup, yet i need the name to invoke - so i think, the non-generated schema should stay